### PR TITLE
Development

### PR DESCRIPTION
**Pull request by furkananter**

### CODE REVIEW

**Comment by bsrakts** [comment]
add comment

File: components/Event/Event.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Title from '../Title/Title';
+import style from './Event.module.css';
+const Event = () => {
+  return (
+
+      <section className='w-full p-28 bg-red-400'>
+
+      <div className='flex flex-col justify-center items-start'>
```
Comment (OrhanOzkercin) [inline components/Event/Event.tsx:9]: Indentation needed here

**Review by OrhanOzkercin — COMMENTED** [review]
Good job 👏

**Comment by eraygundogmus** [comment]
LGTM!

Please ensure that you use conventional or semantic commits!

 https://www.conventionalcommits.org/en/v1.0.0/#summary

**Pull request merged**